autograder/autograder/execution_environments/container_network.py

```diff
@@ -793,6 +793,8 @@ def execute_helper(self, containers, script, arguments, logfile):
             self.log_message("ERROR: Could not verify execution mode status.")
             return
 
+        autograding_utils.cleanup_stale_containers(self.untrusted_user)
```

Comment:
Is there any reason this isn't inside a `finally` block with the previous `try`...`except`?

Reply:
Moved.